src/transport/grpc/mod.rs

```diff
@@ -398,9 +398,9 @@ impl Reconnect for Grpc {
 #[async_trait]
 impl Disconnect for Grpc {
     async fn disconnect(mut self) -> Result<(), crate::Error> {
-        Self::detach(self.client, &self.uuid)
-            .await
-            .map_err(|e| e.into())
+        debug!("detaching node {}", self.uuid);
```

Comment:
```suggestion
        info!("detaching node {}", self.uuid);
```

Reply:
Maybe a debug before and an info after if the detach was successful could be helpful while debugging vs on the debug where the RUST_LOG would be higher than debug.